packages/react-components/src/localization/LocalizationProvider.tsx

```diff
@@ -68,6 +71,12 @@ export interface ComponentStrings {
   errorBar: ErrorBarStrings;
   /** Strings for VideoGallery */
   videoGallery: VideoGalleryStrings;
+  /** Strings for FileUploadCards */
+  fileUploadCards: FileUploadCardsStrings;
```

Comment:
These are beta

Reply:
Should they be removed conditionally here?

Reply:
_FileUploadCards component is marked as internal not marked as beta, should we still remove these conditionally?

Reply:
done